src/main/java/org/ibp/api/java/impl/middleware/ontology/VariableServiceImpl.java

```diff
@@ -76,8 +80,18 @@
 	@Autowired
 	private OntologyScaleDataManager ontologyScaleDataManager;
 
+	@Autowired
+	private OntologyVariableService ontologyVariableService;
+
+	@Autowired
+	private TermValidator termValidator;
```

Comment:
You can remove declaration of `termValidator` here as it's declared already in parent class `ServiceBaseImpl`